### PR TITLE
CIV-12806 Go to hearing defendant

### PR DESCRIPTION
**Pull request by paul-pearson**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-12806

### Change description ###

Go to hearing defendant

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by jeswanth-hmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/DashboardNotificationsParamsMapper.java [hunk 1]
```diff
@@ -37,6 +37,9 @@ public Map<String, Object> mapCaseDataToParams(CaseData caseData) {
             params.put("defendantAdmittedAmountPaymentDeadlineEn", DateUtils.formatDate(whenWillThisAmountBePaid));
             params.put("defendantAdmittedAmountPaymentDeadlineCy", DateUtils.formatDate(whenWillThisAmountBePaid));
         }
+        params.put("defendantName", caseData.getRespondent1().getPartyName());
```
[the remark this answers is not in the document]
Comment (paul-pearson) [reply src/main/java/uk/gov/hmcts/reform/civil/service/DashboardNotificationsParamsMapper.java]: Refactored to use `respondent1PartyName` and `applicant1PartyName`.

**Review by m-meulendijks-v1 — APPROVED** [review]

File: src/main/resources/db/migration/V2024_03_08_1658__CIV-12806-go-to-hearing-paid-partial-defendant-scenario.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+/**
+ * Add notification template
+ */
+INSERT INTO dbs.dashboard_notifications_templates (template_name, title_En, title_Cy, description_En, description_Cy
+                                                  ,notification_role)
+VALUES ('Notice.AAA7.ClaimantIntent.GoToHearing.DefPartAdmit.FullDefence.StatesPaid.ClaimantConfirms.Defendant',
+        'Wait for the court to review the case',
+        'Wait for the court to review the case',
+        '<p class="govuk-body">${applicant1PartyName} wants to proceed to court.</p><p class="govuk-body>They rejected your admission of ${admissionPaidAmount} although they accept you have already paid it.</p><p class="govuk-body>If the case goes to a hearing we will contact you with further details.</p><p class="govuk-body><a href="{VIEW_RESPONSE_TO_CLAIM}"  rel="noopener noreferrer" class="govuk-link">View your response</a><br><a target="_blank" href="{VIEW_CLAIMANT_HEARING_REQS}"  rel="noopener noreferrer" class="govuk-link">View the claimant''s hearing requirements</a></p>',
```
Comment (kenneth-hmcts) [inline src/main/resources/db/migration/V2024_03_08_1658__CIV-12806-go-to-hearing-paid-partial-defendant-scenario.sql:17]: Change <p class="govuk-body> to <p class="govuk-body">

File: src/main/resources/db/migration/V2024_03_08_1658__CIV-12806-go-to-hearing-paid-partial-defendant-scenario.sql [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        '<p class="govuk-body">${applicant1PartyName} wants to proceed to court.</p><p class="govuk-body>They rejected your admission of ${admissionPaidAmount} although they accept you have already paid it.</p><p class="govuk-body>If the case goes to a hearing we will contact you with further details.</p><p class="govuk-body><a href="{VIEW_RESPONSE_TO_CLAIM}"  rel="noopener noreferrer" class="govuk-link">View your response</a><br><a target="_blank" href="{VIEW_CLAIMANT_HEARING_REQS}"  rel="noopener noreferrer" class="govuk-link">View the claimant''s hearing requirements</a></p>',
```
Comment (kenneth-hmcts) [inline src/main/resources/db/migration/V2024_03_08_1658__CIV-12806-go-to-hearing-paid-partial-defendant-scenario.sql:18]: Change "govuk-body to "govuk-body"

Comment (paul-pearson) [reply src/main/resources/db/migration/V2024_03_08_1658__CIV-12806-go-to-hearing-paid-partial-defendant-scenario.sql:18]: Done

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**